chalicelib/app.py

```diff
@@ -344,6 +378,38 @@ def delete_environment(environ):
         return app_utils_obj.forbidden_response()
 
 
+# dmichaels/2022-07-31:
```

Comment:
Do we have a way to test that these will only be accessible to people with appropriate permissions?